_posts/2021-01-06-how-to-update-to-cis-13.adoc

```diff
@@ -0,0 +1,213 @@
+== Deployment walkthrough
+
+=== Step 1: Update references to the Gruntwork Infrastructure as Code Library
+
+In order to update to the CIS AWS Foundations Benchmark v1.3.0, you need to update your references to the Gruntwork
+Infrastructure as Code Library to use compatible versions. We (Gruntwork) have reviewed and updated all the modules in the
+library for compatibility with the new version of the Benchmark. As a customer, you need to update to
+the proper versions of the Gruntwork library to pick up the fixes/changes that were made to be compatible. Refer to
+https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#updating[the
+"Updating" section of "How to use the Gruntwork Infrastructure as Code Library"] for instructions on how to update the
+versions in your code.
+
+For the vast majority of the repos, the only change that will be necessary is a version number bump, but several repos
+require more extensive code changes and state migrations. To upgrade without downtime and data loss, **you MUST follow
+the migration instructions in the release notes in each repo to know what changes need to be made to update to the new
+version.**
+
+[.exceptional]
+IMPORTANT: Gruntwork follows
+https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#versioning[semantic
+versioning]. For any pre-1.0 modules, this means that version updates to the minor version are considered backwards
+incompatible releases for any version updates prior to 1.0.0 release. Make sure to read the release notes for the
+relevant modules any time you are updating minor versions! Note that you will want to read the release notes for each
+minor version that is updated (e.g., if you are going from v0.5.x to v0.9.x, you will want to read the notes for v0.6.0,
+v0.7.0, v0.8.0, and v0.9.0 to get the full list of backwards incompatible updates).
+
+The following table provides a summary of all the relevant Gruntwork AWS modules and the respective versions that are
+compatible with CIS AWS 1.3.0:
+
+[[compatibility_table]]
+[cols="1,1h,1"]
+|===
+|Gruntwork Repo |Minimum version with CIS AWS 1.3.0 support |Corresponding CIS AWS 1.3.0 recommendations
+
+|module-security
+|https://github.com/gruntwork-io/module-security/releases/tag/v0.44.6[v0.44.6]
+|1.20, 1.21, 2.1.1, 2.1.2, 3.10, 3.11
+
+|terraform-aws-monitoring
+|https://github.com/gruntwork-io/terraform-aws-monitoring/releases/tag/v0.24.0[v0.24.0]
+|1.20, 2.1.1, 2.1.2
+
+|package-zookeeper
+|https://github.com/gruntwork-io/package-zookeeper/releases/tag/v0.8.0[v0.8.0]
+|1.20, 2.1.1, 2.1.2
+
+|terraform-aws-vpc
+|https://github.com/gruntwork-io/terraform-aws-vpc/releases/tag/v0.12.0[v0.12.0]
+|1.20, 2.1.1, 2.1.2
+
+|package-openvpn
+|https://github.com/gruntwork-io/package-openvpn/releases/tag/v0.13.0[v0.13.0]
+|1.20, 2.1.1, 2.1.2
+
+|cis-compliance-aws
+|https://github.com/gruntwork-io/cis-compliance-aws/releases/tag/v0.9.2[v0.9.2]
```

Comment:
I think this should be 0.9.3? https://github.com/gruntwork-io/cis-compliance-aws/releases/tag/v0.9.3